src/radical/entk/appman/appmanager.py

```diff
@@ -270,9 +270,28 @@ def run(self):
 
             self._prof.prof('amgr run started', uid=self._uid)
 
+
+            # Submit resource request if not resource allocation done till now or
+            # resubmit a new one if the old one has completed
+            if self._resource_manager:
+                res_alloc_state = self._resource_manager.get_resource_allocation_state()
+                if (not res_alloc_state) or (res_alloc_state in self._resource_manager.get_completed_states()):
```

Comment:
Brackets around the logical expressions are not needed, and line is too long. This can be written as:
```
if not res_alloc_state or \
    res_alloc_state in self._resource_manager.get_completed_states():
```

Reply:
fixed